user/inc/Arduino.h

```diff
@@ -4,6 +4,7 @@
 
 #define PARTICLE_ARDUINO_COMPATIBILITY
 #include "Particle.h"
+#include "spark_wiring_arduino.h"
```

Comment:
This change may break existing libraries, since when original Arduino libraries were ported to Particle, Arduino.h was typically replaced with application.h or Particle.h.  I would suggest leaving #include "spark_wiring_arduino.h" in application.h and keeping the removal of #ifdef PARTICLE_ARDUINO_COMPATIBILITY in spark_wiring_arduino.h so that arduino is just supported by default.